test/new-e2e/tests/otel/utils/config_utils.go

```diff
@@ -42,27 +43,22 @@ func TestOTelFlare(s OTelTestSuite, providedCfg string, fullCfg string, sources
 
 	s.T().Log("Getting latest flare")
 	flare, err := s.Env().FakeIntake.Client().GetLatestFlare()
-	require.NoError(s.T(), err, "Failed to get latest flare")
-	otelFolder, otelFlareFolder := false, false
-	var otelResponse string
-	for _, filename := range flare.GetFilenames() {
-		if strings.Contains(filename, "/otel/") {
-			otelFolder = true
-		}
-		if strings.Contains(filename, "/otel/otel-flare/") {
-			otelFlareFolder = true
-		}
-		if strings.Contains(filename, "otel/otel-response.json") {
-			otelResponse = filename
-		}
-	}
-	assert.True(s.T(), otelFolder)
-	assert.True(s.T(), otelFlareFolder)
-	otelResponseContent, err := flare.GetFileContent(otelResponse)
-	s.T().Log("Got flare otel-response.json", otelResponseContent)
 	require.NoError(s.T(), err)
+	otelflares := fetchFromFlare(s.T(), flare)
+	assert.Contains(s.T(), otelflares, "otel/otel-response.json")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/customer.cfg")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/env.cfg")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/environment.json")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/runtime.cfg")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/runtime_override.cfg")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/health_check/dd-autoconfigured.dat")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/pprof/dd-autoconfigured_debug_pprof_heap.dat")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/pprof/dd-autoconfigured_debug_pprof_allocs.dat")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/command.txt")
+	assert.Contains(s.T(), otelflares, "otel/otel-flare/ext.txt")
+
```

Comment:
Looks like you are missing zpages, it should contain:
- otel/otel-flare/zpages/dd-autoconfigured_debug_extensionz.dat
- otel/otel-flare/zpages/dd-autoconfigured_debug_featurez.dat
- otel/otel-flare/zpages/dd-autoconfigured_debug_pipelinez.dat
- otel/otel-flare/zpages/dd-autoconfigured_debug_servicez.dat
- otel/otel-flare/zpages/dd-autoconfigured_debug_tracez.dat

Also you are missing for pprof:
- otel/otel-flare/pprof/dd-autoconfigured_debug_pprof_profile.dat

Reply:
Those are not present in flares in the tests. Does it need any prerequisites? like some specific configs

Reply:
You can see all the otel flares in the test: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/683936374 search `Got otel flare`

Reply:
NVM, I got the wrong link and now I can see them

Reply:
added them all